packages/db/src/system.ts

```diff
@@ -47,12 +46,10 @@ export type ConnectOptions = Meta.ConnectOptions<Collections>;
 
 const system = Meta.forDefinitions(definitions);
 
-export const connect: (
-  options: TruffleConfig | ConnectOptions | undefined
-) => Db = system.connect;
+export const connect: (options: ConnectOptions | undefined) => Db =
+  system.connect;
 
-export const serve: (
-  options: TruffleConfig | ConnectOptions | undefined
-) => ApolloServer = system.serve;
+export const serve: (options: ConnectOptions | undefined) => ApolloServer =
```

Comment:
(Same as above, re: optionals)

```suggestion
export const serve: (options?: ConnectOptions) => ApolloServer =
```